### PR TITLE
Dịch "softmax-regression" - Phần 3

### LINKED ISSUE #701: Dịch "softmax-regression" - Phần 3
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_linear-networks/softmax-regression_vn.md#L99-L155

### PR DESCRIPTION
**Pull request by NhatQuanBui**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #701 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [x] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_linear-networks/softmax-regression_vn.md [hunk 1]
```diff
@@ -117,7 +117,12 @@ In our case, since we have 4 features and 3 possible output categories, we will
 We compute these three *logits*, $o_1, o_2$, and $o_3$, for each input:
 -->
 
-*dịch đoạn phía trên*
+Để đo lường các xác suất có điều kiện liên quan với các lớp với nhau, chúng ta cần một mô hình với nhiều đầu ra, mỗi đầu ra cho mỗi lớp.
```
Comment (tanduong) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Để tính xác suất có điều kiện ứng với mỗi lớp, chúng ta cần một mô hình với nhiều kết quả, mỗi kết quả cho   một lớp.
```

Comment (duythanhvn) [reply chapter_linear-networks/softmax-regression_vn.md]: @tanduong Bạn giúp mình chỉ comment khi review, hạn chế sử dụng tính năng `Request change` khi chưa cần thiết nhé.

File: chapter_linear-networks/softmax-regression_vn.md [hunk 2]
```diff
@@ -104,9 +104,9 @@ In our case, $y$ would be a three-dimensional vector, with $(1, 0, 0)$ correspon
 
 <!--
 ### Network Architecture
--->
+-->
```
Comment (lkhphuc) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
-->
```

File: chapter_linear-networks/softmax-regression_vn.md (hunk 1 above)
Comment (lkhphuc) [reply chapter_linear-networks/softmax-regression_vn.md]: Mình thấy từ `kết qủa` không rõ nghĩa lắm.
```suggestion
Để tính xác suất có điều kiện ứng với mỗi lớp, chúng ta cần một mô hình có nhiều đầu ra, với một đầu ra cho mỗi lớp.
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 3]
```diff
@@ -117,7 +117,12 @@ In our case, since we have 4 features and 3 possible output categories, we will
 We compute these three *logits*, $o_1, o_2$, and $o_3$, for each input:
 -->
 
-*dịch đoạn phía trên*
+Để đo lường các xác suất có điều kiện liên quan với các lớp với nhau, chúng ta cần một mô hình với nhiều đầu ra, mỗi đầu ra cho mỗi lớp.
+Để phần loại với các mô hình tuyến tính, chúng ta cần nhiều hàm tuyến tính như chúng ta có nhiều đầu ra.
```
Comment (lkhphuc) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Để phân loại với các mô hình tuyến tính, chúng ta cần số hàm tuyến tính nhiều như số đầu ra.
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 4]
```diff
@@ -132,15 +137,18 @@ We can depict this calculation with the neural network diagram shown in :numref:
 Just as in linear regression, softmax regression is also a single-layer neural network.
 And since the calculation of each output, $o_1, o_2$, and $o_3$, depends on all inputs, $x_1$, $x_2$, $x_3$, and $x_4$,
 the output layer of softmax regression can also be described as fully-connected layer.
--->
+-->
```
Comment (lkhphuc) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
-->
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 5]
```diff
@@ -132,15 +137,18 @@ We can depict this calculation with the neural network diagram shown in :numref:
 Just as in linear regression, softmax regression is also a single-layer neural network.
 And since the calculation of each output, $o_1, o_2$, and $o_3$, depends on all inputs, $x_1$, $x_2$, $x_3$, and $x_4$,
 the output layer of softmax regression can also be described as fully-connected layer.
--->
+--> 
 
-*dịch đoạn phía trên*
+Chúng ta có thể mô tả phép tính này với biểu đồ mạng nơ-ron được thể hiện trong : numref:`fig_softmaxreg`.
+Như hồi quy tuyến tính, hồi quy softmax cũng là một mạng nơ-ron đơn tầng.
+Và vì sự tính toán của mỗi đầu ra, $o_1, o_2$, and $o_3$, phụ thuộc vào tất cả đầu vào, $x_1$, $x_2$, $x_3$, and $x_4$,
```
Comment (lkhphuc) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Và vì sự tính toán của mỗi đầu ra, $o_1, o_2$, and $o_3$, phụ thuộc vào tất cả đầu vào, $x_1$, $x_2$, $x_3$, và $x_4$,tầng đầu ra của hồi quy softmax cũng có thể được xem như một tầng fully-connected.
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 6]
```diff
@@ -132,15 +137,18 @@ We can depict this calculation with the neural network diagram shown in :numref:
...
+tầng đầu ra của hồi quy softmax có thể cũng được mô tả như là tầng fully-connected
```
Comment (lkhphuc) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 7]
```diff
@@ -150,8 +158,10 @@ Note that we have gathered all of our weights into a $3\times4$ matrix and that
 our outputs are given by a matrix-vector product of our weights by our inputs plus our biases $\mathbf{b}$.
 -->
 
-*dịch đoạn phía trên*
-
+Để biểu diễn mô hình gọn hơn, chúng ta có thể sử dụng ký hiệu đại số tuyến tính.
+Trong dạng thức vector, chúng ta bắt đầu tại $\mathbf{o} = \mathbf{W} \mathbf{x} + \mathbf{b}$, một dạng thức tốt hơn phù hợp cho cả toán học và cả viết mã nguồn.
```
Comment (lkhphuc) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Trong dạng thức vector, chúng ta có được $\mathbf{o} = \mathbf{W} \mathbf{x} + \mathbf{b}$, một dạng thức phù hợp hơn cho cả viết toán và viết mã. 
```
`arrive at` + đích đến nhé, không phải bắt đầu đâu.

File: chapter_linear-networks/softmax-regression_vn.md (hunk 1 above)
Comment (tanduong) [reply chapter_linear-networks/softmax-regression_vn.md]: > @tanduong Bạn giúp mình chỉ comment khi review, hạn chế sử dụng tính năng `Request change` khi chưa cần thiết nhé.

Bạn định nghĩa thế nào là cần thiết nhé. Mình thấy một số lỗi khá nghiêm trọng. Anyway, mình sẽ comment thôi.

File: chapter_linear-networks/softmax-regression_vn.md (hunk 3 above)
Comment (tanduong) [reply chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Để phân loại bằng mô hình tuyến tính thì ta cần một hàm tuyến tính cho mỗi đầu ra.
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 8]
```diff
@@ -134,13 +139,15 @@ And since the calculation of each output, $o_1, o_2$, and $o_3$, depends on all
 the output layer of softmax regression can also be described as fully-connected layer.
 -->
 
-*dịch đoạn phía trên*
+Chúng ta có thể mô tả phép tính này với biểu đồ mạng nơ-ron được thể hiện trong : numref:`fig_softmaxreg`.
+Như hồi quy tuyến tính, hồi quy softmax cũng là một mạng nơ-ron đơn tầng.
+Và vì sự tính toán của mỗi đầu ra, $o_1, o_2$, and $o_3$, phụ thuộc vào tất cả đầu vào, $x_1$, $x_2$, $x_3$, và $x_4$,tầng đầu ra của hồi quy softmax cũng có thể được xem như một tầng fully-connected.
```
Comment (aivivn) [inline chapter_linear-networks/softmax-regression_vn.md]: từ `fully-connected` đã có trong glossary rồi.
```suggestion
Và vì sự tính toán của mỗi đầu ra, $o_1, o_2$, and $o_3$, phụ thuộc vào tất cả đầu vào, $x_1$, $x_2$, $x_3$, và $x_4$,tầng đầu ra của hồi quy softmax cũng có thể được xem như một tầng kết nối đầy đủ.
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 9]
```diff
@@ -150,8 +157,10 @@ Note that we have gathered all of our weights into a $3\times4$ matrix and that
 our outputs are given by a matrix-vector product of our weights by our inputs plus our biases $\mathbf{b}$.
 -->
 
-*dịch đoạn phía trên*
-
+Để biểu diễn mô hình gọn hơn, chúng ta có thể sử dụng ký hiệu đại số tuyến tính.
+Trong dạng thức vector, chúng ta có được $\mathbf{o} = \mathbf{W} \mathbf{x} + \mathbf{b}$, một dạng thức phù hợp hơn cho cả viết toán và viết mã.
```
Comment (aivivn) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Ở dạng vector, ta có $\mathbf{o} = \mathbf{W} \mathbf{x} + \mathbf{b}$, một dạng phù hợp hơn cho cả toán và lập trình.
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 10]
```diff
@@ -150,8 +157,10 @@ Note that we have gathered all of our weights into a $3\times4$ matrix and that
 our outputs are given by a matrix-vector product of our weights by our inputs plus our biases $\mathbf{b}$.
 -->
 
-*dịch đoạn phía trên*
-
+Để biểu diễn mô hình gọn hơn, chúng ta có thể sử dụng ký hiệu đại số tuyến tính.
+Trong dạng thức vector, chúng ta có được $\mathbf{o} = \mathbf{W} \mathbf{x} + \mathbf{b}$, một dạng thức phù hợp hơn cho cả viết toán và viết mã. 
+Chú ý rằng chúng ta đã tập hợp tất cả các trọng số của mình vào một ma trận $3\times4$ và với một ví dụ có trước $\mathbf{x}$,
```
Comment (aivivn) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Chú ý rằng chúng ta đã tập hợp tất cả các trọng số vào một ma trận $3\times4$ và với một mẫu cho trước $\mathbf{x}$, các đầu ra được tính bởi tích ma trận-vector của các trọng số và đầu vào cộng với vector hệ số điều chỉnh $\mathbf{b}$.
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 11]
```diff
@@ -150,8 +157,10 @@ Note that we have gathered all of our weights into a $3\times4$ matrix and that
 our outputs are given by a matrix-vector product of our weights by our inputs plus our biases $\mathbf{b}$.
 -->
 
-*dịch đoạn phía trên*
-
+Để biểu diễn mô hình gọn hơn, chúng ta có thể sử dụng ký hiệu đại số tuyến tính.
+Trong dạng thức vector, chúng ta có được $\mathbf{o} = \mathbf{W} \mathbf{x} + \mathbf{b}$, một dạng thức phù hợp hơn cho cả viết toán và viết mã. 
+Chú ý rằng chúng ta đã tập hợp tất cả các trọng số của mình vào một ma trận $3\times4$ và với một ví dụ có trước $\mathbf{x}$, 
+các đầu ra của chúng ta được lấy bởi tích số một ma trận vector của các trọng số đầu vào cộng với các độ chệch $\mathbf{b}$.
```
Comment (aivivn) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 12]
```diff
@@ -117,7 +117,12 @@ In our case, since we have 4 features and 3 possible output categories, we will
 We compute these three *logits*, $o_1, o_2$, and $o_3$, for each input:
 -->
 
-*dịch đoạn phía trên*
+Để tính xác suất có điều kiện ứng với mỗi lớp, chúng ta cần một mô hình có nhiều đầu ra, với một đầu ra cho mỗi lớp.
+Để phân loại với các mô hình tuyến tính, chúng ta cần số hàm tuyến tính nhiều như số đầu ra.
+Mỗi đầu ra sẽ tương ứng với hàm tuyến tính của chính nó.
+Trong trường hợp này, vì chúng ta có 4 đặc trưng và 3 loại đầu ra, chúng ta sẽ cần 12 số vô hướng để thể hiện các trọng số,
+($w$ với các chỉ số dưới) và 3 số vô hướng để thể hiện các độ chệch ($b$ với các chỉ số dưới).
```
Comment (tiepvupsu) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Trong trường hợp này, vì có 4 đặc trưng và 3 đầu ra, chúng ta sẽ cần 12 số vô hướng để thể hiện các trọng số, ($w$ với các chỉ số dưới) và 3 số vô hướng để thể hiện các hệ số điều chỉnh ($b$ với các chỉ số dưới).
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 13]
```diff
@@ -117,7 +117,12 @@ In our case, since we have 4 features and 3 possible output categories, we will
...
+Chúng ta sẽ tính ba *logits*, $o_1, o_2$, and $o_3$ , cho mỗi đầu vào:
```
Comment (tiepvupsu) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Chúng ta sẽ tính ba *logits*, $o_1, o_2$, và $o_3$, cho mỗi đầu vào:
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 14]
```diff
@@ -134,13 +139,15 @@ And since the calculation of each output, $o_1, o_2$, and $o_3$, depends on all
 the output layer of softmax regression can also be described as fully-connected layer.
 -->
 
-*dịch đoạn phía trên*
+Chúng ta có thể mô tả phép tính này với biểu đồ mạng nơ-ron được thể hiện trong : numref:`fig_softmaxreg`.
+Như hồi quy tuyến tính, hồi quy softmax cũng là một mạng nơ-ron đơn tầng.
+Và vì sự tính toán của mỗi đầu ra, $o_1, o_2$, and $o_3$, phụ thuộc vào tất cả đầu vào, $x_1$, $x_2$, $x_3$, và $x_4$,tầng đầu ra của hồi quy softmax cũng có thể được xem như một tầng kết nối đầy đủ.
```
Comment (tiepvupsu) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Và vì sự tính toán của mỗi đầu ra, $o_1, o_2$, và $o_3$, phụ thuộc vào tất cả đầu vào, $x_1$, $x_2$, $x_3$, và $x_4$, tầng đầu ra của hồi quy softmax cũng có thể được xem như một tầng kết nối đầy đủ.
```
